Apps/WebClient/src/ClientApp/src/views/ReleaseNotesView.vue

```diff
@@ -31,6 +31,19 @@ export default class ReleaseNotesView extends Vue {
     <div>
         <BreadcrumbComponent :items="breadcrumbItems" />
         <page-title title="Health Gateway Release Notes" />
+        <release-note
+            date="January 18, 2023"
+            version="v2023.1.18"
+            title="Hospital Visits"
+        >
+            <ul>
+                <li>
+                    We added hospital visit information, which includes
+                    admiission and discharge dates, location and provider for
```

Comment:
Typo: "admiission" should be "admission".

Reply:
Good catch!